CraftTweaker2-MC1120-Main/src/main/java/crafttweaker/mc1120/entity/attribute/expand/ExpandEntityAttributeModifier.java

```diff
@@ -0,0 +1,36 @@
+package crafttweaker.mc1120.entity.attribute.expand;
+
+import crafttweaker.annotations.ZenRegister;
+import crafttweaker.api.entity.attribute.IEntityAttributeModifier;
+import crafttweaker.api.minecraft.CraftTweakerMC;
+import net.minecraft.entity.ai.attributes.AttributeModifier;
+import stanhebben.zenscript.annotations.Optional;
+import stanhebben.zenscript.annotations.ZenExpansion;
+import stanhebben.zenscript.annotations.ZenGetter;
+import stanhebben.zenscript.annotations.ZenMethodStatic;
+
+import java.util.UUID;
+
+@ZenExpansion("crafttweaker.entity.AttributeModifier")
+@ZenRegister
+public class ExpandEntityAttributeModifier {
+    private static AttributeModifier getInternal(IEntityAttributeModifier expanded) {
+        return CraftTweakerMC.getAttributeModifier(expanded);
```

Comment:
Technically, you could get NPEs when using this result, if `expanded` was null, but since you'd get the same behavior for dereferencing a null object if this was an instance method, that's fine.